spec.html

```diff
@@ -25000,7 +24999,7 @@
         <emu-alg>
           1. Let _x_ be ? thisNumberValue(*this* value).
           1. Let _f_ be ? ToInteger(_fractionDigits_). (If _fractionDigits_ is *undefined*, this step produces the value 0.)
-          1. If _f_ &lt; 0 or _f_ &gt; 20, throw a *RangeError* exception. However, an implementation is permitted to extend the behaviour of `toFixed` for values of _f_ less than 0 or greater than 20. In this case `toFixed` would not necessarily throw *RangeError* for such values.
+          1. If _f_ &lt; -20 or _f_ &gt; 100, throw a *RangeError* exception.
```

Comment:
I'm not familiar with a mathematical operation involving strings.

It certainly has a well-defined meaning in this PR - but I think that `(24).toFixed(-3)` being anything but an error is confusing and makes no sense, whereas I think `(24).toFixed(-2)` being `0` makes perfect sense but is useless, and `(24).toFixed(-1)` being `20` makes perfect sense and is intuitive with the behavior of `toFixed` given positive numbers (although I still don't see the value or use case of providing a negative number).